scriptworker/test/test_gpg.py

```diff
@@ -590,19 +591,24 @@ def test_rebuild_gpg_home_signed(context, trusted_email, tmpdir):
 
 
 # get_git_revision, get_latest_tag {{{1
-# TODO these two tests should use a tarball with a test git repo inside
 @pytest.mark.asyncio
 async def test_get_git_revision():
-    parent_dir = os.path.dirname(__file__)
+    tar = tarfile.open(os.path.join(os.path.dirname(__file__), "data", "test_git_repo.tgz"))
+    tar.extractall(os.path.join(os.path.dirname(__file__), "data"))
+    parent_dir = os.path.join(os.path.dirname(__file__), "data", "testrepo")
     expected = subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=parent_dir)
```

Comment:
I think a hardcoded sha would be good.

However, the other comment I have is you should use the [`tmpdir` fixture](https://github.com/mozilla-releng/scriptworker/blob/f4475ca/scriptworker/test/__init__.py#L238-L246).

To do this, you would change the function definition to

```
async def test_get_git_revision(tmpdir):
```

pytest will then automatically call the function with the path of a temporary directory that will get cleaned up at the end of the test.  Then you could `tar.extractall` inside of `tmpdir` instead of in `./data`.  You don't have to worry about running `shutil.rmtree` because pytest will clean up `tmpdir` for you after each test run.